### ISSUE TITLE
No way to override defaultTheme.js of devportal

### ISSUE THREAD

**Issue by chanaka3d** [wso2/product-apim#6733]
Followed the following.

# Override public folder
If you want to override a certain devportal webapp static files, this is the correct place to do it.
You do not have to copy the entire directory, only copy the desired file/files.

#### Example

Following will override the main.css file.
```sh
override
    ├── Readme.txt
    └── css
        └── main.css

```

### Configurations
Add following configurations to jaggery.conf file in order to apply the changes.

```json
      "filters":[
        {
            "name":"URLMappingFilter",
            "class":"org.wso2.carbon.apimgt.impl.filter.URLMappingFilter"
        }
      ]


     "filterMappings":[
        {
            "name":"URLMappingFilter",
            "url":"/site/public/*"
        }
     ]
```
Following error is printed in the developer console.

<img width="1680" alt="Screen Shot 2019-10-30 at 11 36 24 AM" src="https://user-images.githubusercontent.com/3424539/67833207-0a25b980-fb0a-11e9-99b6-f0b565d34ca3.png">

**Comment by dushaniw** [comment]
We will not fix this in upcoming 3.1.0 release. Will consider this for future 3.x releases.

**Comment by chanaka3d** [comment]
This requirement is no longer valid. The recommended way to replace these static files is to add them to the CI/CD pipeline.
